bluetooth/src/androidLibMain/kotlin/BluetoothMonitor.kt

```diff
@@ -49,16 +50,17 @@ actual class BluetoothMonitor internal constructor(
         get() = bluetoothAdapter?.isEnabled == true
 
     override fun startMonitoring() {
-        if (bluetoothAdapter == null) return
+        if (bluetoothAdapter == null) warn("BluetoothMonitor") { "bluetoothAdapter is null" }
         applicationContext.registerReceiver(
             availabilityBroadcastReceiver,
             IntentFilter(BluetoothAdapter.ACTION_STATE_CHANGED)
         )
+        updateEnabledState()
     }
 
     override fun stopMonitoring() {
-        if (bluetoothAdapter == null) return
```

Comment:
This check was to avoid call of unregisterReceiver if there is no Bluetooth supported